protonvpn_cli/utils.py

```diff
@@ -321,7 +321,7 @@ def check_root():
     if getpass.getuser() != "root":
         print(
             "[!] The program was not executed as root.\n"
-            "[!] Plase run as root"
+            "[!] Please run as root"
         )
         logger.debug("Program was executed as root")
```

Comment:
I also just noticed that this should say `... wasn't executed as root` for debugging.

If you want you can add this, otherwise I'll do it later :P